models/weatherUpdater.js

```diff
@@ -0,0 +1,81 @@
+const Weather = require("./weatherSchema");
+const weatherApi = require("./weather-api");
+
+module.exports.updateWeather = function(weathers) {
```

Comment:
How about changing name of the function into **saveWeathers** and the file name into **weather-utils**?